pkg/server/serverpb/admin.proto

```diff
@@ -563,8 +563,8 @@ message DataDistributionResponse {
   // By database name.
   map<string, DatabaseInfo> database_info = 1 [(gogoproto.nullable) = false];
 
-  // By zone config id.
-  map<int64, ZoneConfig> zone_configs = 2 [(gogoproto.nullable) = false];
```

Comment:
Can reserve this field number and make a new field if folks think that's important.